ci/experimental/docker/connector_test/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# We use manylinux1 base image because pyarrow_manylinux2010 has a bug and wheel failed to be audited
```

Comment:
This isn't true anymore
```suggestion
```

Reply:
Again, I think this could straight into `ci/docker/connector_test_aarch/Dockerfile`